app/models/actor.rb

```diff
@@ -0,0 +1,5 @@
+class Actor < ApplicationRecord
+  has_attached_file :avatar, styles: { medium: "300x300>", thumb: "100x100>" }, default_url: "/images/:style/missing.png"
```

Comment:
seeing so many avatars, it would have been better to move them to some Attachment model and associate models with it...but leave for now, but do keep in mind for future.